src/renderer/src/components/emulator-pane/use-emulator-video-stream.ts

```
@@ -0,0 +1,210 @@
+        chunkData = new Uint8Array(configBytes.length + data.length)
+        chunkData.set(configBytes, 0)
+        chunkData.set(data, configBytes.length)
+        configBytes = null
+      }
+      // decode() can throw synchronously (DataError/InvalidStateError on malformed
+      // wire bytes); the async error callback won't catch it, so surface via fatal().
+      try {
+        decoder.decode(
+          new ChunkCtor({
+            type: msg.keyFrame ? 'key' : 'delta',
+            timestamp: (timestamp += 1),
```

Biome (2.5.0)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)